docs/platform/release-notes/tizen-6-5-m1.md

```diff
@@ -1,554 +1,565 @@
-  - The ability to support TensorFlow Lite 2 with GPU and NNAPI delegations along with gRPC/flatbuffer with sync and async mode has been added.
-  - The ability to designate priorities between multiple frameworks in case multiple frameworks support the same model file format (for example, TensorFlow-lite’s .tflite format) has been added.
-  - The ORC acceleration for tensor manipulation and subplugin auto detection based on the extension of the model file has been added.
-  - An ml_nnfw_type_e enumeration has been added to support PyTorch and NNtrainer.
-- Machine Learning Training API update
-  - Number of ml_train_layer_type_e has been added.
-    - ML_TRAIN_LAYER_TYPE_BN
-    - ML_TRAIN_LAYER_TYPE_CONV2D
-    - ML_TRAIN_LAYER_TYPE_FLATTEN
-    - ML_TRAIN_LAYER_TYPE_ACTIVATION
-    - ML_TRAIN_LAYER_TYPE_ADDITION
-    - ML_TRAIN_LAYER_TYPE_CONCAT
-    - ML_TRAIN_LAYER_TYPE_MULTIOUT
-- ONE runtime update
-  - ONE runtime has been upgraded from 1.9.0 to 1.15.0.
-  - ONERT uses ARM Compute Library v21.02.
-  - Expanded CPU backend support
-    - 16x1 block sparsity support in Float32 FullyConnected operation.
-    - Add, AvgPool2d, Concat, Conv2D, DepthToSpace, DepthwiseConv2D, Dequantize, Div, Elu, ExpandDims, LeakRelu, LogicalAnd, Maximum, MaxPool2D, Minimum, Mul, Pad, PadV2, Quantize, Rank, Reshape, Resizebiliear, Shape, Softmax, Squeeze, and Sub support for INT8 quantized model
-    - Dequantize, Rank support for UINT8 quantized model
-  - Expanded ACL(ARM Compute Library) backend support
-    - ArgMin support of both CL and NEON backends for Float32, UINT8, INT8 models
-    - ArgMax support of NEON backend for INT8 models
-- TensorFlow Lite 2 with GPU Delegate
-  - TensorFlow Lite 2 (2.3.0) has been added. The GPU Delegate option has been enabled.
-  - To support the backward compatibility of the previous TensorFlow Lite Model, both TensorFlow Lite (1.13) and TensorFlow Lite 2 (2.3.0) have been supported.
-
-#### Fixes
-
-- Machine Learning Inference API
-  - The hidden memory issues such as double free, memory leakage, and so on have been fixed.
-
-#### Known Issues
-
-- TensorFlow Lite 2 with GPU Delegation
-  - To enable GPU Delegation, installation of the OpenCL library has been required.
+# Tizen 6.5 Public M1
+
+Release date: May 31, 2021
+
+
+## Release Details
+
+- [Getting source code](http://review.tizen.org/git/) (Tizen 6.5 M2 source codes are under **tizen** branch.)
+
+- Getting binaries and images
+  - Base: [http://download.tizen.org/releases/milestone/tizen/base/tizen-base_20210503.2](http://download.tizen.org/releases/milestone/tizen/base/tizen-base_20210503.2)
+  - Profile(Unified): [download.tizen.org/releases/milestone/tizen/unified/tizen-unified_20210513.3/](http://download.tizen.org/releases/milestone/tizen/unified/tizen-unified_20210513.3/)
```

Comment:
Minor comment. "http://" is omitted in the Profile link.
![image](https://user-images.githubusercontent.com/9958374/120127506-7ee73b00-c1fa-11eb-8869-45a55295cdac.png)


If you want to keep current, I'll accept.